src/Adapter/Order/CommandHandler/UpdateProductInOrderHandler.php

```diff
@@ -249,4 +275,92 @@ private function assertProductCanBeUpdated(
             }
         }
     }
+
+    /**
+     * @param Cart $cart
+     * @param Order $order
+     * @param OrderInvoice|null $invoice
+     *
+     * @throws \PrestaShopDatabaseException
+     * @throws \PrestaShopException
+     */
+    private function updateCartRules(Cart $cart, Order $order, ?OrderInvoice $invoice = null)
+    {
+        $computingPrecision = Context::getContext()->getComputingPrecision();
```

Comment:
Since the precision depends on the currency, so depends on the cart's currency in BO side you should get precision from it Like in https://github.com/PrestaShop/PrestaShop/blob/develop/src/Adapter/Order/CommandHandler/AddProductToOrderHandler.php#L697